hips/hip-9999.md

```diff
@@ -0,0 +1,80 @@
+
+- attach labels to issues
+- assign issues to milestones
+- receive access to the cncf-helm-core-maintainers mailing list
+- vote on governance-related topics
+- review and approve pull requests
+- merge pull requests
+
+The following roles and resposibilities are NOT allowed to be performed under this role:
+
+- release new revisions of the Helm Client
+- operate any public-facing services, such as
+   - <https://get.helm.sh>
+   - Mailing lists
+   - Slack channels
+   - The helm/helm Github project
+- organizing official LF events (Helm Summit)
+
+These restrictions can be waived if these responsibilities are performed under direct supervision of a Helm Project Maintainer. For example, if a Helm Triage Maintainer would like to learn the release process, they may "shadow" the Helm Project Maintainer that is assigned as the release engineer. Triage Maintainers may also join the Helm Summit Program Committee to help review CFPs, but cannot be assigned the role of Program Chair.
+
+The same rules about active maintainership applies to triage maintainers. Triage maintainers MUST remain active on the project. If they are unresponsive for > 3 months they will lose maintainership unless a super-majority of the other project maintainers agrees to extend the period to be greater than 3 months.
+
+## Backwards compatibility
+
+Community members may still request for a vote to join as a full project maintainer. This new role provides an additional option to join the project at a lower commitment level.
+
+## Security implications
+
+The role helps protect the Helm Client and its users from new maintainers with malicious intent to de-rail or otherwise "pwn" the project. As Helm is one of the CNCF's most popular projects, a certain level of care and caution should be taken when providing new members access to production services. This new role helps keep the list of maintainers with production system access to a minimum.
+
+## How to teach this
+
+The contents of this HIP will be discussed in the public Helm Dev meeting, which is recorded.
+
+## Rejected ideas
+
+The author considered whether a triage maintainer could perform the duties of a release engineer, but rejected as a release engineer needs detailed information about the project's design, intents, and implementation. These tenets are not necessarily required to be a triage maintainer.
+
+## Open Questions
+
+All of the Helm project's secret credentials are stored in the helm_project keybase team, which we currently use for cross-communication between other project maintainer groups. This is a shared team - every project maintainer (helm, helm-www, chartmuseum, etc.) has access to the credentials in that account.
+
+Should those credentials be moved elsewhere so we can invite triage maintainers to the channel, or should triage maintainers be restricted to the mailing list/slack channels?
```

Comment:
Probably best use the mailing list/slack channels for this role. As the triage maintainer is hopefully a stepping stone to project maintainer, then that role would require access to keybase more.

Reply:
OKay. I'd agree with that.